addons/mail/models/mail_mail.py

```diff
@@ -345,7 +345,7 @@ def _send(self, auto_commit=False, raise_exception=False, smtp_session=None):
                     notif_msg = _('Error without exception. Probably due do concurrent access update of notification records. Please see with an administrator.')
                     notifs.sudo().write({
                         'notification_status': 'exception',
-                        'failure_type': 'UNKNOWN',
+                        'failure_type': 'error',
```

Comment:
I actually liked "unknown" more.
"error" is not really a failure type, and is a bit redundant with 'notification_status': 'exception', don't you think?

Reply:
Seems legit.